extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -390,7 +399,12 @@ export class ConsentUI {
    * Enter the fullscreen state for the UI
    */
   enterFullscreen_() {
-    if (!this.ui_ || !this.isVisible_ || this.isFullscreen_) {
+    if (
+      !this.ui_ ||
+      !this.isVisible_ ||
+      this.isFullscreen_ ||
+      this.lightboxEnabled_
```

Comment:
In the case lightbox is enabled, should we send an error message back to the iframe?

Reply:
Yeah, I'm ok with that. More transparency.